code/modules/projectiles/updated_projectiles/guns/rifles.dm

```diff
@@ -345,42 +345,41 @@
 
 
 //-------------------------------------------------------
-//M41AE2 HEAVY PULSE RIFLE
+//T-42 Light Machine Gun
 
 /obj/item/weapon/gun/rifle/standard_lmg
-	name = "\improper M41AE2 heavy pulse rifle"
```

Comment:
Copy this into a new gun, something like `/obj/item/weapon/gun/rifle/m41ae2_hpr`
The repath was to allow for this easily.